commcare_connect/templates/opportunity/add_visits_existing_users.html

```diff
@@ -0,0 +1,113 @@
+{% extends "base.html" %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block title %}{{ request.org }} - {{ opportunity.name }}{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <h1 class="display-5">Edit Opportunity</h1>
+    <h4>Adding Visits for Existing Users</h4>
+    <form method="post"
+          x-data="{
+            max_visits: 0,
+            end_date: '',
+            selected_users: [],
+            isSelected (id) { return this.selected_users.find(el => el === id)},
```

Comment:
adding in 'totalVisits' for using in the confirmation message:

```suggestion
            isSelected (id) { return this.selected_users.find(el => el === id)},
            totalVisits() { return this.selected_users.length * this.max_visits },
```

Reply:
Added. https://github.com/dimagi/commcare-connect/pull/116/commits/611be9ab3e522249b619e82f13991ae0655eabf6